docs/technical_reference/unit_models/uv_aop.rst

```diff
@@ -0,0 +1,96 @@
+
+This model also accounts for the energy demand for the disinfection process. The users need to provide Electrical Efficiency per Log Order Reduction (EE/O) to evaluate energy costs.
+
+Ports
+-----
+
+The model provides two ports (Pyomo notation in parentheses):
+
+* Inlet port (inlet)
+* Outlet port (outlet)
+
+Sets
+----
+.. csv-table::
+   :header: "Description", "Symbol", "Indices"
+
+   "Time", ":math:`t`", "[0]"
+   "Phases", ":math:`p`", "['Liq']"
+   "Components", ":math:`j`", "['H2O', 'NDMA', ...]"
+
+Degrees of Freedom and Variables
+--------------------------------
+The UV system includes the state variables from the associated property package, and an outline of these variables are listed below:
+
+.. csv-table::
+   :header: "Description", "Symbol", "Variable Name", "Index", "Units"
+
+   "Inlet temperature", ":math:`T`", "temperature", "[t]", ":math:`\text{K}`"
+   "Inlet pressure", ":math:`P_{in}`", "pressure", "[t]", ":math:`\text{Pa}`"
+   "Outlet pressure", ":math:`P_{out}`", "pressure", "[t]", ":math:`\text{Pa}`"
+   "Mass flowrate of components", ":math:`M_j`", "flow_mass_phase_comp", "[t, 'Liq', j]", ":math:`\text{kg/s}`"
+   "Inlet volumetric flowrate", ":math:`F_{in}`", "flow_vol", "[t]", ":math:`\text{m}^3\text{/s}`"
+
+**NOTE: Variables for 'temperature', 'pressure', 'flow_mass_phase_comp', and 'flow_vol' come from the associated property package as state variables and are accessed via {port_name}.{state_var_name}**
+
+Aside from the inlet feed state variables (i.e., temperature, pressure, component mass flowrates),
+the UV AOP model has at least an additional 5 degrees of freedom that
+the user must specify. The table below gives an outline of these.
+
+.. csv-table::
+   :header: "Description", "Symbol", "Variable Name", "Index", "Units"
+
+   "Inactivation rate coefficient", ":math:`k`", "inactivation_rate", "['Liq', j]", ":math:`\text{m}^2\text{/J}`"
+   "Pseudo-first order rate constant", ":math:`k_0`", "rate_constant", "['Liq', j]", ":math:`\text{s}^{-1}`"
+   "UV dose", ":math:`D`", "uv_dose", None, ":math:`\text{J/}\text{m}^2`"
+   "Average intensity of UV light", ":math:`I`", "uv_intensity", None, ":math:`\text{J/}\text{m}^2\text{/s}`"
+   "Exposure time of UV light", ":math:`t`", "exposure_time", None, ":math:`\text{s}`"
+   "Electricity demand of components", ":math:`E_j`", "electricity_demand_phase_comp", "[t, 'Liq', j]", ":math:`\text{W}`"
+   "Electricity efficiency per log order reduction (EE/O)", ":math:`EE/O_j`", "electrical_efficiency", "[t, 'Liq', j]", ":math:`\text{J/}\text{m}^3`"
+   "Lamp efficiency", ":math:`\eta`", "lamp_efficiency", None, None
```

Comment:
I know we are saying this implementation can be representative of UV-AOP, but I realized that we don't account for any addition of a chemical (oxidant) dose. Is this something that would be excluded from the model (for now) but accounted for in costing?

Reply:
The energy cost associated with AOP should be accounted for by the EE/O. That is to say, users can provide EE/O with respect to pure UV or UV-AOP technologies, then the electricity demand can be evaluated. The effect of H2O2 has been accounted for in the model, in the PR #600, I added another reaction rate constant to represent indirect photolysis process. You are correct we did need a oxident concentration information for costing chemical dose.

Reply:
> I know we are saying this implementation can be representative of UV-AOP, but I realized that we don't account for any addition of a chemical (oxidant) dose. Is this something that would be excluded from the model (for now) but accounted for in costing?
The energy cost associated with AOP should be accounted for by the EE/O. That is to say, users can provide EE/O with respect to pure UV or UV-AOP technologies, then the electricity demand can be evaluated. The effect of H2O2 has been accounted for in the model, in the PR https://github.com/watertap-org/watertap/pull/600, I added another reaction rate constant to represent indirect photolysis process. You are correct we did need a oxident concentration information for costing chemical dose.

Reply:
OK, that's what I thought. Thanks for verifying!